requirements.txt

```diff
@@ -1,3 +1,4 @@
+absl-py=1.2.0
```

Comment:
Might be worth dumping this until `dev-requirements.txt` since it's only needed if you are working on the code

Reply:
Ok

Reply:
Wait, do we have one already set up and I missed it?

Reply:
NHope for some reason git isn't seeing the dev-requirements file so letting someone else do this.